Project.toml

```diff
@@ -10,6 +10,7 @@ FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
 FourierFlows = "2aec4490-903f-5c70-9b11-9bed06a700e1"
 JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
```

Comment:
We don't need to add Plots as a dependency of the package. When one wants to run some of the examples they should load the `examples/Project.toml`. The docs also have Plots as a dependency; see `docs/Project.toml`.

Reply:
Thanks, I hadn't realized those separate `Project.toml` files existed (I'm new to Julia).

Reply:
no worries!